local/config/finna/FeedbackForms.yaml.sample

```diff
@@ -52,7 +52,11 @@
 #      hideSenderInfo (boolean) Hide default sender fields (name & email)
 #
 #   hideRecipientInfo (boolean) Hide recipient info (default = false)
-#
+#   includeBarcode    (boolean) Whether to submit the user's library card barcode along with other form data.
```

Comment:
Olisiko siitä jotain haittaa, jos includeBarcode:a saisi käyttää muillakin formeilla, kunhan muut edellytykset täyttyvät?

Reply:
Näin tiedetään minkä lomakkeiden tallennuksissa esiintyy tunnuksia. Ylimääräinen varmistus siis, ei muuta syytä.